analytics/src/redux/Join/actions.js

```diff
@@ -0,0 +1,143 @@
+import axios from "axios";
+import setAuthToken from "../../utils/setAuthToken";
+import jwt_decode from "jwt-decode";
+import { GET_ERRORS, SET_CURRENT_USER, USER_LOADING } from "./types";
+// Register User
+export const registerUser = (userData, history) => (dispatch) => {
+  axios
+    .post("http://localhost:3000/api/v1/users/register", userData)
```

Comment:
kinda strange because I switched to using constant variables for the URL strings, I am wondering why this change is not showing up here. Hmmm....